src/encoded/tests/test_schemas.py

```diff
@@ -220,3 +219,48 @@ def test_unique_keys_are_identifying_properties(testapp: TestApp) -> None:
 def has_identifying_property(schema: Dict[str, Any], property_name: str) -> bool:
     """Return True if schema has property_name property."""
     return property_name in schema_utils.get_identifying_properties(schema)
+
+
+def test_submittable(testapp, registry):
```

Comment:
I think this test is a bit redundant with the ones in snovault and will be a minor burden to maintain when things change on the given item type. Consider just grabbing one of the item types from the items with `submitted_id` and checking that the `/submission-schemas/{item_type}.json` endpoint works and looks grossly correct (e.g. lacks all excluded properties, lacks properties with given attributes, etc.) without exact verification of exact properties and property counts.

Reply:
May also be good to check that a non-submittable item fails this endpoint as expected.

Reply:
Added test for expected 404 on non-submittable item.

Regarding the redundancy of the test - I think it's not a bad thing.  However, I take your point on it needing updates if things change.  That was part of why I asked about possible mocking and using a test schema that would not change - or I could certainly simplify as you suggest.  Could go either way.

Reply:
Right, sorry, I missed that on my first time through. My vote is for the second option (using a test schema combined with some snovault mocking) to ensure the project defaults and the endpoint are working as intended.